package.json

```diff
@@ -66,43 +67,46 @@
     ]
   },
   "scripts": {
-    "download:syntax": "ts-node ./build/download-syntax.ts",
-    "preesbuild-base": "npm run download:syntax",
-    "esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --format=cjs --platform=node",
-    "esbuild": "npm run esbuild-base -- --sourcemap",
-    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch",
-    "compile": "npm run esbuild",
-    "watch": "npm run esbuild-watch",
-    "vscode:prepublish": "npm run esbuild-base -- --minify",
-    "package": "vsce package",
     "lint": "eslint src --ext ts",
     "prettier": "prettier \"**/*.+(js|json|ts)\"",
     "format": "npm run prettier -- --write",
     "check-format": "npm run prettier -- --check",
-    "pretest": "npm run compile && npm run lint",
-    "test": "node ./out/test/runTest.js"
+    "download:syntax": "ts-node ./build/download-syntax.ts",
+    "prepare": "npm run download:syntax",
+    "compile": "webpack --mode development",
+    "watch": "webpack --mode development --watch",
+    "prepackage": "npm run download:syntax",
+    "vscode:prepublish": "webpack --mode production --devtool hidden-source-map",
+    "package": "vsce package",
+    "run-in-browser": "vscode-test-web --browserType=chromium --extensionDevelopmentPath=. ."
   },
   "dependencies": {
-    "@vscode/extension-telemetry": "^0.4.9"
+    "@vscode/extension-telemetry": "^0.5.2"
   },
   "devDependencies": {
     "@types/glob": "^7.2.0",
     "@types/mocha": "^9.0.0",
     "@types/node": "^14.18.12",
     "@types/vscode": "^1.61.0",
-    "@typescript-eslint/eslint-plugin": "^5.9.1",
-    "@typescript-eslint/parser": "^5.9.1",
+    "@types/webpack-env": "^1.17.0",
+    "@typescript-eslint/eslint-plugin": "^5.27.0",
+    "@typescript-eslint/parser": "^5.27.0",
     "@vscode/test-electron": "^2.0.3",
-    "esbuild": "^0.14.23",
-    "eslint": "^8.9.0",
+    "@vscode/test-web": "^0.0.24",
+    "assert": "^2.0.0",
+    "eslint": "^8.16.0",
     "eslint-config-prettier": "^8.4.0",
     "eslint-plugin-prettier": "^4.0.0",
     "glob": "^7.2.0",
     "got": "^11.8.3",
     "mocha": "^9.1.3",
     "prettier": "^2.5.1",
+    "process": "^0.11.10",
+    "ts-loader": "^9.3.0",
     "ts-node": "^10.6.0",
-    "typescript": "^4.5.4",
-    "vsce": "^2.6.7"
+    "typescript": "^4.7.3",
+    "vsce": "^2.9.1",
+    "webpack": "^5.72.1",
+    "webpack-cli": "^4.9.2"
```

Comment:
Were all these dependency updates necessary to enable web support? If not - would you mind doing these updates in a separate PR?